tools/topology/topology2/platform/intel/sdw-jack-generic.conf

```diff
@@ -153,7 +153,7 @@ Object.PCM.pcm [
 
 		Object.PCM.pcm_caps.1 {
 			name "volume playback 0"
-			formats 'S16_LE,S32_LE'
+			formats 'S16_LE,S24_LE,S32_LE'
```

Comment:
ok but the name "volume playback 0"  is probably completely wrong, no? Shouldn't this be fixed as well?